fe1-web/src/features/evoting/reducer/__tests__/ElectionReducer.test.ts

```diff
@@ -77,18 +72,13 @@ describe('ElectionReducer', () => {
           [mockElectionState.id]: mockElectionState2,
         },
         allIds: [mockElectionState.id],
+        defaultQuestions: [EMPTY_QUESTION],
       } as ElectionReducerState);
     });
 
     it('throws an error when trying to update an inexistent election', () => {
       expect(() =>
-        electionReduce(
-          {
-            byId: {},
-            allIds: [],
-          } as ElectionReducerState,
-          updateElection(mockElectionState),
-        ),
+        electionReduce(defaultElectionReducerState, removeElection(mockElection.id)),
```

Comment:
is it wanted that the `updateElection` is now `removeElection`?

Reply:
No, I do not know how I did not catch it before. Thanks